package exceptionHandling;

public class ExceptionDemo4 {

	public static void main(String[] args) {
		try {
			System.out.println("vishnu");
		} catch (Exception e) {
			System.out.println("catch block");
		}
			System.out.println("rest of the app");

	}

}
